package.json

```diff
@@ -2,12 +2,14 @@
   "name": "@autorest/csharp",
   "private": true,
   "description": "package.json intended for in-repo use only, package.json used for publishing is located in src/AutoRest.CSharp/package.json",
-  "devDependencies": {
-    "@azure-tools/cadl-ranch-mockapis": "file:test/CadlRanchMockApis",
-    "@azure-tools/typespec-client-generator-core": "0.42.3",
-    "@azure-tools/typespec-csharp": "file:src/TypeSpec.Extension/Emitter.Csharp",
-    "@azure-tools/unbranded-tests": "file:test/UnbrandedProjects",
-    "@microsoft.azure/autorest.testserver": "3.3.48",
-    "autorest": "3.7.1"
-  }
+    "devDependencies": {
+        "@azure-tools/cadl-ranch-mockapis": "file:test/CadlRanchMockApis",
+        "@azure-tools/typespec-azure-rulesets": "0.42.1",
+        "@azure-tools/typespec-azure-resource-manager": "^0.42.1",
```

Comment:
It should be a product dependency instead of devDependency.

Reply:
reverted since they are not needed